commands/iam.js

```diff
@@ -13,7 +14,7 @@ function genCode(n) {
 
 // if email has not been verified, send verification code
 // linked to IAM command
-const iam = async function (userid, email, nickname, affiliation, sgMail) {
+const iam = async function (userid, email, nickname, affiliation) {
   // check email against allowed domains
   let domain = email.match(
     '^[a-zA-Z0-9_.+-]+@(?:(?:[a-zA-Z0-9-]+.)?[a-zA-Z]+.)?(' +
```

Comment:
should this be changed to only accept UCLA email IDs?

Reply:
I believe Rohit has it already set up so that there's an if to check if it includes the .ucla.edu domain in the email